api/node/src/interpreter/component_compiler.rs

```diff
@@ -60,6 +61,16 @@ impl JsComponentCompiler {
             .collect()
     }
 
+    #[napi(setter)]
+    pub fn set_library_paths(&mut self, paths: HashMap<String, String>) {
```

Comment:
IMO there should be a getter for this, as well as a test. This could be done separate from this PR. If yes, please file an issue.

Reply:
It's an preparation to use it from js. But to finish it we need to define the api for js to do it. Also with include_paths. 

Issue:  #3745